### ISSUE TITLE
unit test failed

### ISSUE THREAD

**Issue by beew** [wxMaxima-developers/wxmaxima#1903]
Hi, compiled wxmaxima from source and run the unit test with make test, all tests passed but one 

```
The following tests FAILED:
	180 - testbench_simple.wxmx (SEGFAULT)
```

The error log is attached. WxMaxima 24.02.2 , maxima 5.45.1 using SBCL 2.3.7. OS Ubuntu 22.0.4. Error log is attached.

Thanks.
[LastTest.log](https://github.com/wxMaxima-developers/wxmaxima/files/14657275/LastTest.log)

**Comment by gunterkoenigsmann** [comment]
As far as I can see the program seems to crash on closing after having evaluated all the worksheet. If that is the case this might be #1898 , but only with a single window.

Do you have any means to get a backtrace of this crash?

**Comment by beew** [comment]
Yes, there is a popup window that says "testbench_simple.wxmx has changed , do you want to save it" something like that. There are three options: save, cancel and no. Crashes after clicking anyone. Before the pop up it was running ok.

I don't know how to backtrace and I have deleted the source folder after installation.
